FinderCompass/src/main/java/atomicstryker/findercompass/common/DefaultConfigFilePrinter.java

```diff
@@ -290,8 +292,14 @@ else if (settingList.isEmpty())
 			} else {
 				int[] ids;
 				for(int damage = 0; (ids=OreDictionary.getOreIDs(new ItemStack(pBlock, 1, damage))).length > 0; damage++){

```

Comment:
I have another idea. It is also possible to ignore ore dictionary if -1 is explicitly set as damage. In this case it will only uses the block name. It searchs in ore dictionary without or with damage >= 0.

Please let me know, if it could be fine. In this case I will change the source.